MantidQt/CustomInterfaces/src/Muon/MuonAnalysis.cpp

```diff
@@ -1885,7 +1892,14 @@ void MuonAnalysis::selectMultiPeak(const QString &wsName,
                    std::back_inserter(groupsAndPairs), &QString::fromStdString);
     std::transform(groups.pairNames.begin(), groups.pairNames.end(),
                    std::back_inserter(groupsAndPairs), &QString::fromStdString);
-    m_uiForm.fitBrowser->setAvailableGroups(groupsAndPairs);
+    auto tmp = m_groupingHelper.parseGroupingTable().groupNames;
+    auto names = m_groupingHelper.parseGroupingTable().pairNames;
+    names.insert(std::end(names), std::begin(tmp), std::end(tmp));
+    QStringList GroupsAndPairsNames;
+    for (auto name : names) {
+      GroupsAndPairsNames << QString::fromStdString(name);
+    }
+    m_uiForm.fitBrowser->setAvailableGroups(GroupsAndPairsNames);
```

Comment:
This block of code is duplicated from above - can we move it into a separate function and call it twice instead?